packages/cli/openapi-parser/src/v3/extensions/fernExtensions.ts

```diff
@@ -62,6 +62,20 @@ export const FernOpenAPIExtension = {
      */
     STREAMING: "x-fern-streaming",
 
+    /**
+     * Used to specify if an endpoint should be generated
+     * as a paginated endpoint.
+     *
+     * Example usage:
+     *   paths:
+     *     /path/to/my/endpoint:
+     *       x-fern-pagination:
+     *         page: $request.page
+     *         next: $response.next
+     *         results: $response.results
+     */
+    PAGINATION: "x-fern-pagination",
```

Comment:
should we also support at the root level? 

Then the OpenAPI importer can even detect which endpoints should be paginated automatically. (can do this in a follow-up).